packages/conventional-gitmoji/src/getGitmojiConventional.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/order */
+import _find from 'lodash/find.js'
+import { gitmojis } from 'gitmojis'
```

Comment:
This is a `json` file so will not have type declarations.

Import as a `const require` please.